taxi/views.py

```diff
@@ -87,3 +88,32 @@
 class DriverDetailView(LoginRequiredMixin, generic.DetailView):
     model = Driver
     queryset = Driver.objects.all().prefetch_related("cars__manufacturer")
+
+
+class DriverCreateView(LoginRequiredMixin, generic.CreateView):
+    model = Driver
+    form_class = DriverCreationForm
```

Comment:
The `DriverCreationForm` is correctly used for creating drivers, which aligns with the checklist requirement: 'Use `UserCreationForm` while creating user.'